src/view/screens/Home.tsx

```diff
@@ -70,6 +74,10 @@ function HomeScreenReady({
   const rawSelectedFeed = useSelectedFeed()
   const setSelectedFeed = useSetSelectedFeed()
   const maybeFoundIndex = allFeeds.indexOf(rawSelectedFeed as FeedDescriptor)
+  /*
+   * N.B. if `rawSelectedFeed` returns `DEFAULT_FEED_DESCRIPTOR`,
+   * `maybeFoundIndex` will be -1 and we'll fall back to left-most tab
+   */
```

Comment:
See usage here. We pass `setSelectedFeed ` `FeedDescriptor`s, which can vary for the primary algo. So if we want to select the primary algo, we have this handling to fall back to index 0 (left-most tab).